Bio/codonalign/codonalignment.py

```diff
@@ -98,18 +91,7 @@ def __getitem__(self, index, alphabet=None):
         elif isinstance(col_index, int):
             return "".join(str(rec[col_index]) for rec in self._records[row_index])
         else:
-            from Bio.Alphabet import generic_nucleotide
-
-            if alphabet is None:
-                return MultipleSeqAlignment(
-                    (rec[col_index] for rec in self._records[row_index]),
-                    generic_nucleotide,
-                )
-            else:
-                return MultipleSeqAlignment(
-                    (rec[col_index] for rec in self._records[row_index]),
-                    generic_nucleotide,
```

Comment:
That if/else looked broken as it didn't use the alphabet?

Reply:
Yes